From 32aa9352d8def3d616b50f197c3910e3bd802ac6 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:08:11 -0400
Subject: [usb] appledisplay: Convert /n to n

Message-id: <1410887463-20674-420-git-send-email-dzickus@redhat.com>
Patchwork-id: 95570
O-Subject: [RHEL7 PATCH 419/591] USB: appledisplay: Convert /n to \n
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit e2b86c1d39d03f566514160d2e8b63e4645095d0
Author: Joe Perches <joe@perches.com>
Date:   Thu Apr 24 18:51:02 2014 -0700

    USB: appledisplay: Convert /n to \n

    Use a newline character appropriately.

    Signed-off-by: Joe Perches <joe@perches.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/misc/appledisplay.c b/drivers/usb/misc/appledisplay.c
index ba6a5d6..f37c78d 100644
--- a/drivers/usb/misc/appledisplay.c
+++ b/drivers/usb/misc/appledisplay.c
@@ -110,7 +110,7 @@ static void appledisplay_complete(struct urb *urb)
    __func__, status);
   return;
  default:
-  dev_dbg(dev, "%s - nonzero urb status received: %d/n",
+  dev_dbg(dev, "%s - nonzero urb status received: %d\n",
    __func__, status);
   goto exit;
  }
-- 
1.7.1